radio/src/pulses/pulses_arm.h

```diff
@@ -180,4 +180,28 @@ inline void SEND_FAILSAFE_1S()
   }
 }
 
+#define LEN_R9M_POWER_VALUES           "\006"
+
+#if defined(MODULE_D16_EU_ONLY_SUPPORT)
+  #define TR_R9M_POWER_VALUES          "25 mW\0" "500 mW"
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove the whitespaces before the # character. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3APPDirectiveIndentation)